### PR TITLE
fix: support multi listener, and update readme.md

### PR DESCRIPTION
**Pull request by dk-lockdown**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by jjeffcaii — APPROVED** [review]
LGTM

**Review by Penglq — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -4,16 +4,33 @@
 ![](./docs/pics/arana-logo.png)
 
 ## Introduction | [中文](https://github.com/dubbogo/arana/blob/master/docs/README_ZH.md)
-hello, db.
+
+Arana is a db proxy. It can be deployed as a sidecar, also can be deployed standalone.
```
Comment (AlexStocks) [inline README.md:8]: Arana is a db proxy. It can be deployed as a sidecar, also can be deployed standalone as a proxy.

**Review by Lvnszn — APPROVED** [review]

**Review by bohehe — APPROVED** [review]
LGTM

**Pull request merged**